### DISCUSSION TITLE
Allow to reserve disk space in advance

### DISCUSSION THREAD

**Discussion by pyk** [supabase/supabase#12806] [Feature Requests]
Hey, I want to migrate some production data around 30GB to a new supabase project. Currently I need to wait another 6 hours for disk expansion and It's very inconvenience to wait customer service 24-48 hours to expand your disk. I would love to see a "reserve disk in advance" feature to speed up data migration in the future. thank you!

**Comment by MildTomato** [comment]
This is being actively worked on 👍 

It will be available for Pro tier and above.
